lib/log.c

```diff
@@ -65,8 +65,5 @@ _log (log_level level, const char *file, unsigned lineno, const char *fmt, ...)
     /* Print the user supplied message */
     vfprintf (stream, fmt, argptr);
 
-    /* always add a new line so the user doesn't have to */
-    fprintf (stream, "\n");
```

Comment:
@tstruk thanks a lot for your feedback. In fact I thought about that option too, but at the end I decided not to do it because:

* I couldn't come with a good name for the macros that wouldn't add the new line.
* Other output printing functions that I'm familiar with (i.e: printk, pr_* and dev_* in Linux) don't add a new-line char to the string and let the developer to device. 
* There's a lot of places in tpm2-tools where LOG_* macros are called with a new-line char manually added to the string. So I wouldn't say that the interface is very obvious, it seems that many people expects a printf-like behavior.

So I thought that more macros would just add more confusion to the developers for little gain.

Having said that, I don't have a strong opinion on this so I'll be happy to do what you suggest if people decide that's the better approach.